apps/codeforafrica/src/payload/collections/Media.js

```diff
@@ -0,0 +1,24 @@
+const Media = {
+  slug: "media",
+  admin: {
+    useAsTitle: "alt",
+  },
+  access: {
+    read: () => true, // Everyone can read Media
+  },
+  upload: {
+    staticURL: "/media",
+    staticDir: "media",
+  },
+  fields: [
+    {
+      name: "alt",
+      label: "Alt Text",
+      type: "text",
+      localized: true,
```

Comment:
Will codeforAfrica site support multiple languages? If not, no need for localization